### PR TITLE
Fix #8 General Localization - README

### LINKED ISSUE #8: studyGroup pages breaking with "HTTPS Everywhere"
… which is understandable (and I chose to show "non-secure" content), but is there a way to prevent this from happening?

I've tentatively disabled `HTTPS Everywhere` on Firefox for GitHub Pages.

### PR DESCRIPTION
**Pull request by WW-Web-Infra**
@auremoser The README-ch.md is in Chinese Language. Please review it!

### CODE REVIEW

File: README-ch.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Mozilla 研究组
+
+欢迎使用 Mozilla 科学实验的研究小组的项目! 在这里, 我们将为您进行设置以满足您的需要开始您自己的研究组。
+
+## 等待, 什么是 “Mozilla 研究组” 吗?
+
+Mozilla 的研究组是有趣的、非正式的 meetups, 你的朋友和同事们从您当地的机构或城共享技能、案例和思想上使用的代码的研究和探索开放的研究实践。 我们的目标是要建立一种友好的、没有压力-环境中人们可以共享他们的工作, 请求帮助关于编码的问题, 一起学习并与他们的同事。 任何人都可以启动一项研究组-- 你不一定要是专家编码器要这样做!
+
+什么做的研究组在实际生活吗? 检查了波士顿大学的研究组的网站和大学的 Tornonto 编码器的网站。 您也可以观看几个短的视频从研究组的领导在我们的方向指南。
```
Comment (auremoser) [inline README-ch.md]: This should be Toronto (like the city in Canada), I updated the english readme with correct spelling too!

File: _config.yml [hunk 2]
```diff
@@ -28,7 +28,7 @@ headlineMain: "Let's Work Together"
 # 2. change 'USER' in the example urls below to the username of the google account you made for your calendar
 # 3. set calendar_on: True.
 
-calendar_on: False
+calendar_on: True
 calendar_embed_url: "https://www.google.com/calendar/embed?src=USER@gmail.com"
```
Comment (auremoser) [inline _config.yml]: Would you mind removing this file from your pull request? I think you mean to updated your own calendar on your fork, and not the calendar from the studyGroup base repo. Unfortunately, when you submit a pull request, it chains subsequent commits to that PR until your PR is merged. Sorry!

File: README-ch.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Mozilla 研究组
```
Comment (auremoser) [inline README-ch.md]: This is so cool. I don't speak chinese so I can't verify the content, do you have a friend who can proofread and give a 👍 ?

**Review by auremoser — CHANGES_REQUESTED** [review]
Thanks so much for this, it's awesome, I asked for one file to be removed and minor changes.

**Comment by WW-Web-Infra** [comment]
@auremoser Thank you for the review! Review for Chinese language is difficult! Hindi is the second largest language spoken by Indians! So, I have made the Readme in Hindi! I know Hindi Language (Reading and Writing). If you are fine then we can merge it!

**Review by auremoser — APPROVED** [review]
Looks great, I'm sorry I don't read hindi so I can't QA this :/ but it looks great for now! Thank you so much!

**Review by auremoser — APPROVED** [review]
Great, thank you!

File: README.md [hunk 4]
```diff
@@ -1,5 +1,3 @@
-
-
```
Comment (auremoser) [inline README.md]: Would be great to add "Read this in Hindi" as a link here, just like the "read this in Portuguese" link below this line.

**Review by auremoser — CHANGES_REQUESTED** [review]
If you would add the link to the readme for Hindi here, that would be lovely. Thank you!

**Comment by WW-Web-Infra** [comment]
@auremoser surely will add that and update it :)

**Review by auremoser — APPROVED** [review]
Excellent, thank you!!

**Comment by elionaimc** [comment]
Great job done with hindi language and updating the general readme.md file.
Two questions:
a) Dont think that is better delete the file finishing with ~ character? (it seems like used to be a temporary file)
b) With last update in the general readme.md file, it looks very diferent of the portuguese version, do you people think it is ok leave as is or change it to make a real translation of the new english version?

**Comment by auremoser** [comment]
Hey @elionaimc ! 
@AnamikaD did the translation 🎊  to her!
> a) Dont think that is better delete the file finishing with ~ character? (it seems like used to be a temporary file)

Yes, usually, what file are you referencing? I try to update the `.gitignore` file with unwanted commit files

> b) With last update in the general readme.md file, it looks very diferent of the portuguese version, do you people think it is ok leave as is or change it to make a real translation of the new english version?

Good point, yes we changed the README in English and never updated the Portuguese equivalent for lack of experience with Portuguese. I'll add it as an issue to update the Portuguese readme.

Thank you for your comments!

**Comment by auremoser** [comment]
I just removed the hidden file, thanks @elionaimc !
And I added an issue #76 for the Portuguese translation!

**Comment by elionaimc** [comment]
Great! 
I will start working with README portuguese translation right now!

**Pull request merged**